Gemfile

```diff
@@ -2,6 +2,10 @@ source 'https://rubygems.org'
 
 gemspec
 
+platforms :ruby_19 do
+  gem 'json', '~> 1.8'
+end
+
```

Comment:
@sferik Then I'd rather just drop 1.9 from Travis/AppVeyor, I think it's officially end of life anyway.